src/config-as-code/repo-config-service.ts

```diff
@@ -0,0 +1,55 @@
+import YAML from "yaml";
```

Comment:
Once the structure has settled a bit, let's also create a test for this file (and add a test case for the new function in the GitHub client)